docs/root-application.md

```diff
@@ -56,16 +56,17 @@ promise must be resolved with the child application. The most common implementat
 `() => System.import('/path/to/child-application.js')`
 
 ### Activity function
-The third argument to `declareChildApplication` must be a function that takes in no arguments and returns a truthy
+The third argument to `declareChildApplication` must be a function, the function is provided `window.location` as the first argument, and returns a truthy
```

Comment:
`window.location` does not evaluate to a string. Yet in the tests below it looks like you are passing a string to this function. I'm confused 😕